api/tests/unit/domain/usecases/create-organization-invitations_test.js

```diff
@@ -19,27 +18,24 @@ describe('Unit | UseCase | create-organization-invitations', function () {
       const organizationId = 1;
       const emails = ['member@organization.org'];
       const locale = 'fr-fr';
```

Comment:
**chipotage** : retirer du titre de ce test le role dans should `create one organization-invitation with organizationId, role and email`